### PR TITLE
140 update and increase redability of documentation

### PR DESCRIPTION
**Pull request by expeditionengineer**

### CODE REVIEW

File: webcentral/doc/06_sphinx/source/general/getting_started.md [hunk 1]
```diff
@@ -1,33 +1,36 @@
...
 ```
     ./run build_initial dev
 ```
-3. In the `Wissensplattform` images like logos of in the database included tools, are not located in the repository. These files have to be downloaded from the following link `https://tubcloud.tu-berlin.de/f/3546499069`. The `media`-folder has to be copied to `01_application/webcentral_app/`.
-3. Start the setup process by calling the run script with the argument `up_initial` and providing a database dump file.
+To interact with the web application a bash script with the name `run` is used. This script is used to interact with the application.  
+
+4. In the `Wissensplattform` images like logos of in the database included tools, are not located in the repository. These files have to be downloaded from the following link `https://tubcloud.tu-berlin.de/f/3546499069`. The `media`-folder has to be copied to `webcentral/src/media/`.
+
+5. In a last step the database needs to be populated with data. For that a database dump is located inside the repository under `postgres/webcentral_db_20240927_translation_use_cases_data_sufficiency.sql`.
```
Comment (c0nb4) [inline webcentral/doc/06_sphinx/source/general/getting_started.md:23]: Maybe we can just link to the folder, then we don't need to upgrade this path from time to time.

File: webcentral/doc/06_sphinx/source/general/getting_started.md [hunk 2]
```diff
@@ -1,33 +1,36 @@
...
+```{warning}
+Please note that the filename of the database dump may change. Please look for the latest file with the extension `.sql` in the `postgres/`-folder.
+```
+
+Start the setup process by calling the run script with the argument `up_initial` and providing a database dump file.
+```
```
Comment (c0nb4) [inline webcentral/doc/06_sphinx/source/general/getting_started.md:29]: See previous comment

Comment (expeditionengineer) [reply webcentral/doc/06_sphinx/source/general/getting_started.md:29]: I have removed the filename of the sql file and added a generic sql-filename in the `up_inital` command

**Review by c0nb4 — APPROVED** [review]
LGTM

**Pull request merged**